.github/workflows/artifacts.yml

```
@@ -88,6 +88,10 @@ jobs:
         if: github.event_name != 'release'
         run: npx nx affected --target=docker-build
 
+      - name: Tag all docker images on main also as latest
+        if: github.ref_name != 'main'
```

There is a typo here actually (my error):

```suggestion
        if: github.ref_name == 'main'
```

---

Ah, I should pay more attention when copy/pasting :see_no_evil: